examples/websockets/v2_example.py

```diff
@@ -20,7 +23,8 @@ async def print_trade_update(tu):
 def main():
     logging.basicConfig(level=logging.INFO)
     feed = 'iex'  # <- replace to SIP if you have PRO subscription
-    stream = Stream(data_feed=feed, raw_data=True)
+    
+    stream = Stream(data_feed=feed, raw_data=True)  # <- add base_url=URL('https://paper-api.alpaca.markets') for paper
```

Comment:
I would wrap it like this:
```suggestion
stream = Stream(
        data_feed=feed,
        # base_url='https://paper-api.alpaca.markets', # uncomment for paper
        raw_data=True)
```